data-structures/ds-cque/java/CircularQueue.java

```diff
@@ -0,0 +1,126 @@
+public class CircularQueue
+{
+  int SIZE = 5;
+  int front, rear;
+  int items[] = new int[SIZE];
+
+    CircularQueue ()
+  {
+    front = -1;
+    rear = -1;
+  }
+
+  // Check if the queue is full
+  boolean isFull ()
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6358132)